### ISSUE TITLE
Grooming proposal - for Project Board

### ISSUE THREAD

**Issue by JJediny** [18F/tts-tech-portfolio#314]
## Background information

[ProjectBoard.md](https://github.com/18F/tts-tech-portfolio/blob/master/ProjectBoard.md)

[Radiating Intent: Grooming & Sprint Consideration](https://gsa-tts.slack.com/archives/GP559GCLD/p1578415424014800)

[Radiating Intent: Grooming Labels](https://gsa-tts.slack.com/archives/GP559GCLD/p1578085991010000)

## User stories

As a member of TTS Tech Portfolio, I want everyone to be on the same page about the people, process and tools for the Grooming ceremony. 

## Implementation

- [ ] Make a pull request to the [ProjectBoard.md](https://github.com/18F/tts-tech-portfolio/blob/master/ProjectBoard.md) 
   - [ ] with the suggested changes for Grooming Labels
   - [ ]  when grooming occurs outside of ceremonies
       - [ ] What grooming should be done prior to Planning Ceremony
       - [ ] What grooming should be done prior to Grooming Ceremony

## Acceptance criteria

- [ ] Add grooming methods
- [ ] Demo updates at next Review

**Comment by afeld** [comment]
Closing in favor of https://github.com/18F/tts-tech-portfolio/issues/356.
